doc/theory/glossary.tex

```diff
@@ -167,26 +167,35 @@
 	
 	\subsection*{Persistent simplicial (co)homology:} \label{persistent simplicial (co)homology}
 	
-	Let $\{X(s)\}_{s \in \mathbb R} $ be a collection of ordered or directed simplicial complexes together with simplicial maps $f_{st} : X(s) \to X(t)$ for each pair $s \leq t$, such that if $r \leq s \leq t$, then $f_{rt} = f_{st} \circ f_{rs}$. Its \textit{persistent simplicial homology with $\Bbbk$-coefficients} is the persistence module
+	Let $\{X(s)\}_{s \in \mathbb R} $ be a set of ordered or directed simplicial complexes together with simplicial maps $f_{st} : X(s) \to X(t)$ for each pair $s \leq t$, such that 
+	\begin{equation*}
+	r \leq s \leq t\ \quad\text{implies} \quad f_{rt} = f_{st} \circ f_{rs}
+	\end{equation*}
+	for example, a \hyperref[filtered complex]{filtered complex}. Its \textit{persistent simplicial homology with $\Bbbk$-coefficients} is the persistence module
 	\begin{equation*}
 	H_*(X(s); \Bbbk)
 	\end{equation*}
-	with structure maps $H_*(f_{st}) : H_*(X(s); \Bbbk) \to H_*(X(t); \Bbbk)$ induced form the maps $f_{st.}$ In general, the collection constructed this way needs not satisfy the tameness condition of a \hyperref[persistence module]{persistence module}, but we restrict attention to the cases where it does.
-	
-	\textit{Persistence simplicial cohomology with $\Bbbk$-coefficients} is defined analogously.
+	with structure maps $H_*(f_{st}) : H_*(X(s); \Bbbk) \to H_*(X(t); \Bbbk)$ induced form the maps $f_{st.}$ In general, the collection constructed this way needs not satisfy the tameness condition of a \hyperref[persistence module]{persistence module}, but we restrict attention to the cases where it does. Its \textit{persistence simplicial cohomology with $\Bbbk$-coefficients} is defined analogously.
 	
-	\subsection*{Vietoris-Rips complex and Vietoris-Rips homology:} \label{vietoris-rips complex and vietoris-rips homology}
+	\subsection*{Vietoris-Rips complex and Vietoris-Rips persistence:} \label{vietoris-rips complex and vietoris-rips persistence}
 	
-	The \textit{Vietoris-Rips complex} of a \hyperref[finite metric spaces and point clouds]{finite metric space} $(X, d)$ is given by the following construction: let $s \geq 0$, define the {\hyperref[abstract simplicial complex]{abstract simplicial complex}} $VR_X(s)$ to have vertices the set $X$ and declare a subset $\{x_0, \dots, x_n\}$ of distinct points in $X$ to be a simplex if $d(x_i, x_j) \leq s$ for all $x_i, x_j$, explicitly
+	Let $(X, d)$ be a \hyperref[finite metric spaces and point clouds]{finite metric space}. Define the Vietoris-Rips complex of $X$ as the \hyperref[filtered complex]{filtered complex} $VR_s(X)$ that contains a subset of $X$ as a simplex if all pairwise distances in the subset are less than or equal to $s$, explicitly
```

Comment:
This might not be the right PR to flag this, but are we sure we want to present the VR construction as tied to the metric space category? I was under the impression that the bare structure necessary for a VR complex was simply that of a weighted graph, where the weights must be real but can even be negative. It is true that the metric space view becomes essential when comparing VR to Cech, but would it be too difficult/undesirable to decouple the two aspects?

Btw, in this way we could link VR complexes to flag complexes as defined in the earlier subsection.

Reply:
This is a good point. I think a paragraph explaining the extension beyond metric spaces is should be included. I was going to add a paragraph to the flag entry about something similar and an issue that Guillaume raised. I would want to address this two in that PR if that is OK with you.

Reply:
No problem, let's move this to a PR addressing #340